### ISSUE TITLE
Update dependency with known security vulnerabilities

### ISSUE THREAD

**Issue by jan-kiszka** [siemens/industrialbenchmark#10]
https://github.com/siemens/industrialbenchmark/network/alert/industrial_benchmark_java/pom.xml/com.google.guava:guava/open

Please fix ASAP, it's open for too long now.

**Comment by pswazinna** [comment]
the link is leading nowhere for me - what is this about?

**Comment by jan-kiszka** [comment]
https://github.com/siemens/industrialbenchmark/network/alerts, then click on the warning.

**Comment by jan-kiszka** [comment]
Oh, you may not have the permission. You picked you as main contributor, but only @danielhein84 may see the alert.

**Comment by pswazinna** [comment]
is it possible to change that? he's on holiday rn

**Comment by jan-kiszka** [comment]
As I'm not involved in this project, I'd prefer to not do such a decision on my own. Here is the data from the report:


1 com.google.guava:guava vulnerability found in industrial_benchmark_java/pom.xml on 16 Jun
Remediation

Upgrade com.google.guava:guava to version 24.1.1 or later. For example:
```
<dependency>
  <groupId>com.google.guava</groupId>
  <artifactId>guava</artifactId>
  <version>[24.1.1,)</version>
</dependency>
```
Always verify the validity and compatibility of suggestions with your codebase.
Details
CVE-2018-10237
moderate severity
Vulnerable versions: > 11.0, < 24.1.1
Patched version: 24.1.1

Unbounded memory allocation in Google Guava 11.0 through 24.x before 24.1.1 allows remote attackers to conduct denial of service attacks against servers that depend on this library and deserialize attacker-provided data, because the AtomicDoubleArray class (when serialized with Java serialization) and the CompoundOrdering class (when serialized with GWT serialization) perform eager allocation without appropriate checks on what a client has sent and whether the data size is reasonable.

**Comment by pswazinna** [comment]
updated guava to version 29.0-jre
